instance/testing.py

```diff
@@ -1,33 +1,76 @@
 # -*- coding: utf-8 -*-
-from flask import Markup
 from os import environ
+from flask import Markup
 
-#: The title of this site
 SITE_TITLE = 'Lastuser'
+DEBUG_TB_ENABLED = False
+DEBUG_TB_INTERCEPT_REDIRECTS = False
+SERVER_NAME='test.lastuser.dev:7500'
+LOGFILE = 'error.log'
+SQLALCHEMY_BINDS = {
+    'lastuser': environ.get('SQLALCHEMY_DATABASE_URI', 'postgres://@localhost:5432/lastuser_test_app'),
+}
+SQLALCHEMY_ECHO = False
+SECRET_KEY = 'random_string_here'
+TIMEZONE = 'Asia/Calcutta'
+CACHE_TYPE = 'redis'
+
+#: Use SSL for some URLs
+USE_SSL = False
+
+#: Mail settings
+MAIL_SUPRESS_SEND = True
+MAIL_FAIL_SILENTLY = False
+DEFAULT_MAIL_SENDER = environ.get('DEFAULT_MAIL_SENDER')
+MAIL_DEFAULT_SENDER = DEFAULT_MAIL_SENDER  # For new versions of Flask-Mail
+SITE_SUPPORT_EMAIL = environ.get('SITE_SUPPORT_EMAIL')
+# Mail secrets
+MAIL_SERVER =  environ.get('MAIL_SERVER')
+MAIL_PORT = environ.get('MAIL_PORT')
+MAIL_USE_SSL = environ.get('MAIL_USE_SSL')
+MAIL_USE_TLS = environ.get('MAIL_USE_TLS')
+MAIL_DEFAULT_SENDER = environ.get('MAIL_DEFAULT_SENDER')
```

Comment:
> Looks like the earlier DEFAULT_MAIL_SENDER lines can be removed entirely.
Fixed.

> As this is supposed to be a tuple, we may have to split the value from the environment at the space.
This  should work if the environment variable is set as `Name <email>`?